index.html

```diff
@@ -6,208 +6,234 @@
-                    <p>Languages</p>
-                    </li>
-                      <ul class="media">
-                        <li>JavaScript</li>
-                        <li>Ruby</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                      </ul>
-                </ul>              
-              </div>
+              </ul> 
             </div>
-            </article>
-              <section class="qoute">
-                  <div class="holder-text">
-                  <img src="images/holder.svg" alt="holder">
-                  <div class="holder-text1">
-                    <img src="images/ellipse2.svg" alt="holder">
-                    <ul>
-                      <li class="Language">
-                        <p>Frameworks</p>
-                      </li>
-                      <ul class="media">
-                        <li>React.js</li>
-                        <li>Ruby on Rails</li>
-                        <li>RSpec</li>
-                        <li>Capybara</li>
-                        <li>Selenium</li>
-                      </ul>
-                    </ul>                   
-                    </div>
-                 </div>
-                  </section>
-                  <article class="qoute">
-                    <div class="holder-text">
-                    <img src="images/holder.svg" alt="holder">
-                    <div class="holder-text1">
-                      <img src="images/skills.svg" alt="holder">
-                      <ul>
-                        <li class="Language">
-                          <p>Skills</p>
-                        </li>
-                        <ul class="media">                                                  
-                          <li>Database Management</li>
-                          <li>Version Controle</li>
-                          <li>CLI</li>
-                          <li>WEb Development</li>
-                          <li>API Design</li>
-                        <ul>
-                      </ul>
-                     
-                      </div>
-                    </div>
-                </article>
-              </ul>
-                  </div>
-                  
+          </div>
+        </article>
+      </ul>
+    </div>
+  </section>
+  <section class="contact-form">
+    <div class="contact-container">
+      <p>I'm always interested in hearing about new projects, 
+        so if you'd like to chat please get in touch.</p>
+    </div>
+    <form class="form-inputs" action="https://formspree.io/f/xayzgdak" method="post">
+          <input type="text" name="user_name" maxlength="30" placeholder="Full name" required>
+          <input type="email" name="user_email" placeholder="Email adress" required>  
+          <textarea name="user_message" maxlength="500" placeholder="Hey I'd like to connect to..." required></textarea>
+          <button class="button-text" type="submit">Get in touch</button>
```

Comment:
- [ ] [OPTIONAL] Nice work so far 👍🏿 but I will suggest you implement a good hover effect for this button so as to improve visual accessibility